src/test/java/utils/SeleniumActions.java

```diff
@@ -17,11 +21,37 @@ public static void click(By by, WebDriver driver) {
             } catch (ElementClickInterceptedException | StaleElementReferenceException e) {
                 System.out.println("Can't find it man " + e.getMessage());
                 try {
-                    Thread.sleep(10);
+                    Thread.sleep(3000);
                 } catch (InterruptedException ex) {
                     throw new RuntimeException(ex);
                 }
             }
         }
     }
+
+    public static void clickUntil(By toBeDisplayed, By toClick, WebDriver driver) {
```

Comment:
rename this, and think of other testers. a method name must be descriptive

Reply:
Done